packages/runtime/src/parser/parser.ts

```diff
@@ -960,3 +960,30 @@ export function getProgramVersion(program: string): number {
 	opcodeFromSentence(wordsFromLine(firstLine), 1, ip, ExecutionMode.APPLICATION);
 	return ip.tealVersion;
 }
+
+/**
+ *
+ * @param str string to convert
+ * @returns camelCase to hypenated string eg(fooBar to foo-bar)
+ */
+export function convertCapitalToHyphens(str: string): string {
+	return str.replace(/[A-Z]/g, (m) => "-" + m.toLowerCase());
+}
+
+/**
+ *
+ * @param object
+ * @returns object keys parsed from camelCase to hypenated str eg(fooBar to foo-bar)
+ */
+export function convertKeysToHyphens(object: any) {
```

Comment:
same as above

Reply:
I have explained it in `@returns` part

Reply:
lets move it to the description of the function what it does. In returns part we only say what it returns

Reply:
okay